resources/definitions/ultimaker.def.json

```diff
@@ -120,7 +120,7 @@
         "support_line_distance": { "minimum_value_warning": "0 if support_structure == 'tree' else support_line_width" },
         "support_tower_maximum_supported_diameter": { "value": "support_tower_diameter" },
         "support_tower_roof_angle": { "value": "0 if support_interface_enable else 65" },
-        "support_wall_count": { "value": "1 if support_structure == 'tree' else 0" },
+        "support_wall_count": { "value": "1" },
```

Comment:
I found that changing this setting definitely has a big effect on the jitters. I'm a bit concerned this will make it difficult to see if your engine changes have improved the support. 

At this point though, if it works it works.

Reply:
(You probably already know, just to be sure; this is meant to go with my backend PR!)

It's you can still test with walls off if you like ;-)

I think the engine changes will be _harder_ to see without walls actually! The piece of code we're changing is about the _areas_, not the subsequent infill generation. It really makes it hard to see if the error is yours or if it's just the walls again when its just outright not printing infill in an area for all the layers to the bottom from a certain point onwards.